mlc_llm/quantization/__init__.py

```diff
@@ -52,6 +52,19 @@
         ),
         final_fc_weight="same_as_linear_weight",
     ),
+    "q3f16_1": QuantizationScheme(
+        name="q3f16_1",
+        linear_weight=GroupQuantizationSpec(
+            dtype="float16",
+            mode="int3",
+            sym=True,
+            storage_nbit=16,
+            group_size=40,
+            transpose=False,
+        ),
+        embedding_table="same_as_linear_weight",
```

Comment:
Just curious why quantization spec for embedding table is different for `q3f16_1` and `q3f16_0`?